tests/interfaces/test_batch_torch.py

```diff
@@ -399,7 +399,7 @@ def cost(a):
             dev = qml.device("default.qubit.autograd", wires=2)
             return dev.batch_execute([tape])[0]
 
-        expected = qml.grad(cost)(0.1)
+        expected = qml.grad(cost, argnum=0)(0.1)

```

Comment:
This test was raising a warning wrt. to not marking trainable parameters correctly as per the ongoing deprecation we have.